api/AudioParam.json

```diff
@@ -284,7 +284,7 @@
             "chrome_android": {
               "version_added": "18",
               "partial_implementation": true,
-              "notes": "Does not work."
+              "notes": "This sets the target volume at the specified time, but it doesn’t ramp to it, causing this function to behave like <code>setValueAtTime</code>."
```

Comment:
Please add parenthesis to the function name. So: `setValueAtTime()`. Do this throughout.

Reply:
I still think this is a bug, not a partial implementation issue.  But I defer to @jpmedley for the final word.

For Safari/webkit, it is possible that it is a partial implementation.  Don't know if Safari has kept up with the spec for `exponentialRampToValueAtTime` and friends.

Reply:
@rtoy, to clarify: partial implementation in BCD is a [value indicating whether or not the implementation of the sub-feature deviates from the specification in a way that may cause significant compatibility problems.](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#partial_implementation)  Minor bugs are simply a note without partial implementation set.  This is a bug major enough that causes significant compatibility problems; so, we’re declaring partial implementation and explaining the bug. So no need to worry, we’re on the same page.  😉

@jpmedley Sure thing, coming right up!

Reply:
How's it look now?